cypress/support/commands/navigation.js

```diff
@@ -149,3 +149,29 @@ Cypress.Commands.add('selectRowByColumnAndValue', (columnName, expectedValue, mo
         });
     });
 });
+/**
+ * Select all rows on the current page
+ *
+ */
+Cypress.Commands.add('selectAllRowsOnCurrentPage', () => {
+  cy.get('body').type('{alt}a');
+});
+/**
+ * Select all rows from all pages
+ * This function only works on a list window, and not on a single view window
+ * @param windowId - the window for which the rows will be counted
+ */
+Cypress.Commands.add('countAllRows', windowId => {
```

Comment:
i don't think this is a good use-case.
Why is this function changing the current page that i am on? 
The way i see this function, `windowId` doesn't belong here and i should run `cy.countAllRows();` which returns me all the rows on all pages of a grid view and does nothing more. Imo changing the page is an unexpected (and undesired) side-effect of the function.